zebrad/src/components/mempool/crawler/tests.rs

```diff
@@ -78,29 +76,3 @@ proptest! {
         })?;
     }
 }
-
-/// Create a mock service to represent a [`PeerSet`][zebra_network::PeerSet] and intercept the
-/// requests it receives.
-///
-/// The intercepted requests are sent through an unbounded channel to the receiver that's also
-/// returned from this function.
-fn mock_peer_set() -> (
-    Buffer<BoxService<Request, Response, BoxError>, Request>,
-    UnboundedReceiver<Request>,
-) {
-    let (sender, receiver) = mpsc::unbounded_channel();
-
-    let proxy_service = tower::service_fn(move |request| {
-        let sender = sender.clone();
-
-        async move {
-            let _ = sender.send(request);
-
-            Ok(Response::TransactionIds(vec![]))
-        }
-    });
-
-    let service = Buffer::new(BoxService::new(proxy_service), 10);
-
-    (service, receiver)
-}
```

Comment:
This was just moved to `zebrad/src/components/tests.rs` so that other tests can use it too